.github/workflows/lint-gh-workflows.yml

```diff
@@ -1,6 +1,12 @@
 name: Lint GH Workflows
-on:
-  push:
+
+on: 
+  merge_group:
```

Comment:
Do we want to filter on paths here too like the `pull_request`?

Reply:
merge_group doesnt support that hehe